aiconfig-docs/docs/editor.md

```diff
@@ -0,0 +1,166 @@
+
+:::note
+We also support YAML in addition to JSON for the AIConfig file format.
+:::
+
+To get started, here’s an example prompt chain created in the AIConfig Editor and the corresponding AIConfig. See [Getting Started Tutorial](./getting-started) for details.
+
+![image1](https://github.com/lastmile-ai/aiconfig/assets/129882602/3e291206-a01e-44ad-ac18-c562bdec2555)
+
+This is saved to an [AIConfig JSON file](https://raw.githubusercontent.com/lastmile-ai/aiconfig/main/cookbooks/Getting-Started/travel.aiconfig.json).
+
+## Edit and save AIConfigs {#edit-and-save}
+
+If you already have an AIConfig JSON file, you can use the AIConfig Editor to visually edit the prompts and model parameters.
+
+Open Terminal, run this command:
+
+```bash
+aiconfig edit --aiconfig-path {file_path_existing_aiconfig}
+```
+
+A new tab with the AIConfig Editor opens in your default browser with the prompts, chaining logic, and settings from the specified AIConfig populated in the editor. If the file path doesn’t exist, a new AIConfig will be created at that path and the editor will be blank.
+
+### Saving
+
+Your edits in AIConfig Editor will auto-save and update the AIConfig file every 15 seconds. There is also a `Save` button to manually save changes to your AIConfig.
+
+## Run Prompts {#run-prompts}
+
+Each cell in AIConfig Editor is used to prompt generative AI models and output responses. Editor cell features:
+
+| Feature                          | Description                                                                                                                                                   |
+| -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Prompt Name**                  | The name of the prompt cell which can be referenced in other cells for chaining.                                                                              |
+| **Model**                        | The model you are prompting in this cell. Use the dropdown to see the available default models to AIConfig Editor.                                            |
+| **Settings**                     | The settings and parameters specific to the model (i.e. system prompt, temperature). These settings will vary depending on the model selected.                |
+| **Local Variables (Parameters)** | These are variables that you set to be used in the prompt via handlebars syntax. Local variables are local to the cell and cannot be accessed in other cells. |
+
+Click ▶️ at the bottom right of the cell to execute the prompt and see the model response. Support for streaming responses is coming soon!
```

Comment:
Ryan moved this to the top to prevent the issue where for streaming with variable output cell size it was hard to locate and click to cancel. Just say "on the right side of the cell"

Reply:
We also have streaming responses already so you can delete that line